### ISSUE TITLE
`@Json(naming = Naming.LowerUnderscore)` not work

### ISSUE THREAD

**Issue by kevin70** [avaje/avaje-jsonb#27]
Hello, `@Json` can't use `naming`.
Expected normal adapter generation.

Compilation failed The following is the exception information:
```
java.lang.IllegalArgumentException: No enum constant io.avaje.jsonb.Json.Naming.io.avaje.jsonb.Json.Naming.LowerUnderscore
```

**Comment by rbygrave** [comment]
Pushed up a test that uses LowerUnderscore but it passes - there was no error.

Can you have a look at the test and see if you can see a difference there?

**Comment by rbygrave** [comment]
The problem must be with: https://github.com/avaje/avaje-jsonb/blob/main/jsonb-generator/src/main/java/io/avaje/jsonb/generator/NamingConventionReader.java#L22

The string value of `io.avaje.jsonb.Json.Naming.io.avaje.jsonb.Json.Naming.LowerUnderscore` .. is kind of wrong.

That almost suggests something interesting in the tooling like javac etc. What JDK / javac are you using?

I can probably create a test that simulates that strange string value ...

**Comment by rbygrave** [comment]
Fixed in 1.0-RC2 which I have just released.  You should see 1.0-RC2 in maven central in about 15 minutes.

Can you try that 1.0-RC2 version and confirm that it fixes this issue?  

Thanks, Rob.

**Comment by kevin70** [comment]
OK, I will try to update the version later.
